### PR TITLE
disabled -native-compiler ondemand option due to noise

### LINKED ISSUE #1489: Noise coming from native compiler option
Unfortunately, the `-native-compiler ondemand` option creates a lot of noise when building the library with coq 8.13.2. This was introduced in #1474.

```
Warning: Native compiler is disabled, -native-compiler ondemand option
ignored. [native-compiler-disabled,native-compiler]
```

Currently there is no way to hide this warning but coq/coq#13821 fixes this issue.

I therefore think we should disable these arguments in our `_CoqProject` file and wait till coq 8.14 to re-enable them.

@JasonGross since this mostly affects you, is this the right way to go about it?

### PR DESCRIPTION
**Pull request by Alizter**
closes #1489

### CODE REVIEW

File: _CoqProject [hunk 1]
```diff
@@ -4,7 +4,7 @@
 #-arg -boot
 -arg -noinit
 -arg -indices-matter
--arg -native-compiler -arg ondemand
+#-arg -native-compiler -arg ondemand
```
Comment (SkySkimmer) [inline _CoqProject:7]:
```suggestion
-arg -native-compiler -arg no
```
This should not warn and AFAIK we don't use native at all.

Comment (Alizter) [reply _CoqProject:7]: @JasonGross want's to use native with the library. See #1474 and #1454.

Comment (JasonGross) [reply _CoqProject:7]: I say it's better to leave it disabled for now; I'd prefer `-native-compiler no` to not passing any argument, at least until we make the library compile with the native compiler.  What's the warning, though?

Comment (Alizter) [reply _CoqProject:7]: See #1489

**Comment by Alizter** [comment]
@JasonGross does this look good?

**Review by JasonGross — APPROVED** [review]

**Pull request merged**